cmd/portal/main.go

```diff
@@ -20,6 +20,21 @@ import (
 // injected at link time using -ldflags.
 var version string
 
+// Initialization of cobra and viper.
+func init() {
+	cobra.OnInitialize(initViperConfig)
+
+	rootCmd.PersistentFlags().BoolP("verbose", "v", false, "Specifes if portal logs debug information to a file on the format `.portal-[command].log` in the current directory")
```

Comment:
Specifes -> Specifies

Maybe shorten this to "Log debug information to a file with the format `.portal-[command].log` in the current directory"